README.md

```diff
@@ -582,6 +582,63 @@ Then, import it:
 Generate a nix attribute set from the opam file. This is just a Nix
 representation of the JSON produced by `opam2json`.
 
+### Monorepo functions
+
+#### `queryToMonorepo`
+
+```
+{ repos = ?[Repository]
+; resolveArgs = ?ResolveArgs
+; filterPkgs ?[ ] }
+→ Query
+→ Scope
+```
+
+Similar to `queryToScope`, but creates a attribute set (instead of a
+scope) with package names mapping to sources for replicating the
+[`opam monorepo`](https://github.com/tarides/opam-monorepo) workflow.
+
+The `filterPkgs` argument gives a list of package names to filter from
+the resulting attribute set, rather than removing them based on their
+opam `dev-repo` name.
+
+#### `buildOpamMonorepo'`
+
+```
+{ repos = ?[Repository]
+; resolveArgs = ?ResolveArgs
+; pinDepends = ?Bool
+; recursive = ?Bool
+; extraFilterPkgs ?[ ] }
+→ project: Path
+→ Query
+→ Scope
+```
+
+A convenience wrapper around `queryToMonorepo`, similar to
+`buildOpamMonorepo'`.
+
+Creates a monorepo for an opam project.
+
+The `extraFilterPkgs` argument removed packages by name from the
+resulting attribute set in addition the default filtering of
+ocaml-system, opam-monorepo, and any packages in the query or
+project directory.
+
+#### Examples
+
+Build a monorepo from a local directory:
+
+<div class=example id=build-opam-project-all dir=my-package>
+
+
+
+```nix
+(buildOpamProject' { } ./. { }).my-package
+```
+
+</div>
+
```

Comment:
This whole example is incorrect. It needs a different `id`, also most likely a different `dir` and also the Nix code of the example is wrong.

Reply:
(the new folder should be in `examples/readme`, and should contain the opam-monorepo opam file and so on)

Reply:
Whoops, thanks for pointing that out.

Given that this is a relatively specialist use case that is most likely going to be used from https://github.com/RyanGibb/hillingar I'm happy to leave this without an example project if you are.

Reply:
Hm, ok. Would you consider moving the functionality of `hillingar` to `opam-nix` in a future PR? I'm happy to give you commit access to opam-nix to simplify maintenance.

Reply:
Yep, I would, that sounds like it would be sensible. I created it as a separate project as it's quite mirage specific (and I wasn't sure if there would be an appetite to incorporate it into opam-nix) but also it's still quite volatile as I'm patching it to work for different unikernels. After it's more stable I'll merge create a separate PR.